content/posts/2024-06-03-financial_support.md

```diff
@@ -0,0 +1,54 @@
+Title: Financial_Support
+    Date: 2024-06-03 13:51:49
+    Modified: 2024-06-03 13:51:49
+    Category: announcement
+    Tags: zh-tw, financial-support, PyCon TW 2024
+    Slug: 2024-06-03-financial_support
+    Authors: Simon Huang
+    Summary: PyCon Taiwan 2024 財務補助申請指南
```

Comment:
I think it would be better if we can add a [FA picture](https://media.discordapp.net/attachments/1243967398587928606/1246308349226516530/0603_.png?ex=6661d959&is=666087d9&hm=0e486459749dcf2d9a4578de05e39163a38b66ade1d457bc6a33dd8c2a06f061&) here.
[2024-02-28-pycon-taiwan-2024-cfp-is-open.md](https://github.com/pycontw/pycontw-blog/blob/main/content/posts/2024-02-28-pycon-taiwan-2024-cfp-is-open.md) has an example about attaching pictures, FYR. Thank you Simon~~

Reply:
Note that both summary page and article page need to attach pictures~